cloudsim-plus/src/main/java/org/cloudbus/cloudsim/resources/DatacenterStorage.java

```diff
@@ -0,0 +1,139 @@
+package org.cloudbus.cloudsim.resources;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+
+import org.cloudbus.cloudsim.datacenters.Datacenter;
+import org.cloudbus.cloudsim.util.DataCloudTags;
+
+public class DatacenterStorage {
```

Comment:
Please include small documentation for the class and the tag @author with the name of authors from the DatacenterSimple class, including your own.